test/cli/packages_cmd_test.go

```diff
@@ -209,55 +209,55 @@ func TestRegistryAuth(t *testing.T) {
 	}{
 		{
 			name: "fallback to keychain",
-			args: []string{"packages", "-vv", "registry:localhost:5000/something:latest"},
+			args: []string{"packages", "-vv", "registry:localhost:17/something:latest"},
```

Comment:
nit: Since we have a handful of test cases all using the same value in setups and assertions, it might be a good idea to extract out the common parts as constants (scoped to the `TestRegistryAuth` func), such as `localhost:17` and/or `localhost:17/something:latest`.